### ISSUE TITLE
Fill the blanks fields of inactive card only halfway correct

### ISSUE THREAD

**Issue by Sc0r** [ssricardo/anki-plugins#82]
Ive combined your brilliant addon with a table addon,  but I tested it and it also happens before I convert it to a table. 
following card:
```
Geschäftsfall 			|	USt 	| Ust Wert 	| Netto			| Brutto 		| Konten 		| A/P 		| EA/EE 	| +/ -		| S/H 		| Buchungssatz 	| Soll 			| Haben 
Q/KB: Fachzeitschriften	|{{c1::x}}	|{{c1::x}}	|{{c1::x}}		|{{c1::x}}		|{{c1::Kasse}} 	|{{c1::A}}	|{{c1::x}}	|{{c1::-}}	|{{c1::h}}	|{{c1::Fachz}}	|{{c1::35,05}} 	| {{c1::x}} 	
bar bezahlt	37,5 		|{{c2::7%}}	|{{c2::2,45}} | {{c2::35,05}} |{{c2::37,50}}|{{c2::Fachzeitschriften/Bücher}} |{{c2::x}} |{{c2::EA}} |{{c2::+}}  |{{c2::s}} |{{c2::Vst}}	|{{c2::2,45}}	| {{c2::x}}
						|  {{c1::x}}  |  {{c1::x}}  | {{c1::x}}  |  {{c1::x}}  |{{c1::Vst}}  | {{c1::A}} | {{c1::x}} |  {{c1::+}} | {{c1::s}} |  {{c1::an Kasse}}  	|  {{c1::x}}  | {{c1::37,50}}  	
```
Could it possibly be due to the high number of blanks?

looks like this when it asks me to fill c1
![image](https://user-images.githubusercontent.com/10654417/106156289-a31a0900-6181-11eb-84d1-95ecc3a96429.png)
looks fine again when showing results for c1
![image](https://user-images.githubusercontent.com/10654417/106156448-d492d480-6181-11eb-914a-9a0b4912c451.png)

**Comment by ssricardo** [comment]
Can you Post how your note template is?

Overall it works with fields in multiple cards, I need to check why that happened on this case

**Comment by Sc0r** [comment]
hi there, sorry for the late response.
Must be honest, not quite sure what you mean by note template. Could be called different in the German version of anki. 
I've attached the exported file. It currently happens only with the card where only the top line is being requested to fill. Line 2 is shown correctly, but line 3 for some reason is not.
[Ust.zip](https://github.com/ssricardo/anki-plugins/files/6034860/Ust.zip)

**Comment by ssricardo** [comment]
Solved on new version (2.0).
Let me know if you still get problems
